src/main/resources/migration/common/V37__add_calculation_reason.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE calculation_reason
+(
+    id           INTEGER PRIMARY KEY,
+    rank         INTEGER NOT NULL, -- Determines the order on the page, less than zero will not be displayed.
+    display_name VARCHAR(40),
+    is_other     BOOLEAN DEFAULT FALSE
+);
+
+ALTER TABLE calculation_request
+    ADD COLUMN reason_for_calculation INT REFERENCES calculation_reason (id);
+ALTER TABLE calculation_request
+    ADD COLUMN other_reason_for_calculation VARCHAR(40);
+
+UPDATE calculation_request
+SET reason_for_calculation = -1
+WHERE reason_for_calculation IS NULL;
+
+INSERT INTO calculation_reason (id, rank, display_name)
```

Comment:
I'm not sure why we're ranking them? Shouldn't they just be `active` or `inactive`?

Reply:
My thought was a rank so the order can be preserved in the front end, there may be another method of doing this?